src/py/mat3ra/made/utils.py

```diff
@@ -58,6 +58,25 @@ def get_center_of_coordinates(coordinates: List[List[float]]) -> List[float]:
     return np.mean(np.array(coordinates), axis=0).tolist()
 
 
+def get_overlapping_coordinates(
```

Comment:
Should go to `utils/coordinate.py`

Reply:
It's here to avoid circular imports. This is imported by made/basis.py (for collision detection on `add_atom`) which is outside the made/tools/utils/coordinate.py

Reply:
OK - tracked for future refactoring.